_redirects

```diff
@@ -0,0 +1 @@
+/docs/fanatical-support-aws/access-and-permissions/user-management-and-perms /docs/fanatical-support-aws/user-management-and-perms
```

Comment:
Try this formatting:
```
/docs/fanatical-support-aws/access-and-permissions/user-management-and-perms/*     /docs/fanatical-support-aws/user-management-and-perms/:splat    200
```

Reply:
That does not seem to work either. Kind of seems like netlify is not reading the file.